lib-ee/emqx_ee_bridge/test/emqx_ee_bridge_mysql_SUITE.erl

```diff
@@ -392,3 +410,114 @@ t_write_failure(Config) ->
         end
     ),
     ok.
+
+% This test doesn't work with batch enabled since it is not possible
+% to set the timeout directly for batch queries
+t_write_timeout(Config) ->
+    ProxyName = ?config(proxy_name, Config),
+    ProxyPort = ?config(proxy_port, Config),
+    ProxyHost = ?config(proxy_host, Config),
+    {ok, _} = create_bridge(Config),
+    Val = integer_to_binary(erlang:unique_integer()),
+    SentData = #{payload => Val, timestamp => 1668602148000},
+    Timeout = 10,
+    ?check_trace(
+        emqx_common_test_helpers:with_failure(timeout, ProxyName, ProxyHost, ProxyPort, fun() ->
+            query_resource(Config, {send_message, SentData, [], Timeout})
+        end),
+        fun(Result, _Trace) ->
+            ?assertMatch({error, {resource_error, _}}, Result),
+            ok
+        end
+    ),
```

Comment:
Minor nit: since the trace is not being used, we might just drop `?check_trace`.

```suggestion
    emqx_common_test_helpers:with_failure(timeout, ProxyName, ProxyHost, ProxyPort, fun() ->
        ?assertMatch({error, {resource_error, _}}, query_resource(Config, {send_message, SentData, [], Timeout}))
    end),
```